src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -43,6 +44,27 @@ inline float get_load_time(std::chrono::steady_clock::time_point start_time) {
     return std::chrono::duration_cast<std::chrono::milliseconds>(stop_time - start_time).count();
 }
 
+inline bool ends_with(std::string str, std::string postfix) {
+    return str.rfind(postfix) == str.size() - postfix.size();
+}
+
+std::string get_directory(const std::string& s) {
```

Comment:
this function and one above, are not required.

Reply:
Removed.